test/unittests/libethereum/GasPricer.cpp

```diff
@@ -95,95 +95,95 @@ BOOST_AUTO_TEST_CASE(basicGasPricer_RPC_API_Test_Frontier)
 {
 	u256 _expectedAsk = 16056883295;
 	u256 _expectedBid = 1;
-	dev::test::executeGasPricerTest("RPC_API_Test_Frontier", 30.679, 15.0, "/BlockchainTests/bcGasPricerTest/RPC_API_Test.json", TransactionPriority::Medium, _expectedAsk, _expectedBid, eth::Network::FrontierTest);
+    dev::test::executeGasPricerTest("RPC_API_Test_Frontier", 30.679, 15.0, "/BlockchainTests/ValidBlocks/bcGasPricerTest/RPC_API_Test.json", TransactionPriority::Medium, _expectedAsk, _expectedBid, eth::Network::FrontierTest);
```

Comment:
Would be better to get rid of repetition here, moving `"/BlockchainTests/ValidBlocks/"` to a constant

Reply:
or by moving it inside `executeGasPricerTest` function